lib/chrome/webdriver/builder.js

```diff
@@ -25,6 +25,9 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
     }
 
     const serviceBuilder = new chrome.ServiceBuilder(chromedriverPath);
+    if (options.chrome.chromedriverPort) {
```

Comment:
This need either use lodash.get or you can add a check like:
if (options.chrome  && options.chrome.chromedriverPort) ...

The reason is that even when you run Chrome, we aren't 100% you have set any options for Chrome (that's the way it works right now at least).

Reply:
Yes, sure thing.